package.json

```diff
@@ -22,10 +22,11 @@
     }
   ],
   "engines": {
-    "node": ">=0.12.0"
+    "node": ">=4"
   },
   "scripts": {
     "test": "xo && npm run prepublish && nyc ava",
+    "flow": "flow; test $? -eq 0 -o $? -eq 2",
```

Comment:
@thejameskyle Why `test $? -eq 0 -o $? -eq 2`? Why can't `flow` just have a flag not to exit with code 1 on failure?